test/mpi/rma/get_acc_flush.c

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (C) by Argonne National Laboratory
+ *     See COPYRIGHT in top-level directory
+ */
+#include <stdio.h>
+#include <stdint.h>
+#include <string.h>
+#include <stdlib.h>
+#include <assert.h>
+#include <unistd.h>
+#include <mpi.h>
+#include "mpitest.h"
+
+#define GET_BUF_SIZE 2147467264LL
+#define GET_PER_SIZE 16384
+#define ACC_BUF_SIZE 32768
+#define SIGNATURE 0x77777777
```

Comment:
Can you describe why each GET or ACC has to be this size? Besides, what is `GET_PER_SIZE`?

Reply:
Main idea is to set MPI_Get buffer as large as possible so it will take a long time to complete; the same for accumulate buffer, but in the reversed way. We can use other buffer sizes as well. 
I think I should change `GET_PER_SIZE` to `GET_LAST_SIZE`; it means last MPI_Get data size